### PR TITLE
Fix example and remove false deprecated message

### PR DESCRIPTION
**Pull request by Yang-33**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,6 +26,24 @@ This library requires Go 1.11 or later.
...
+    "github.com/line/line-bot-sdk-go/v7/linebot/httphandler"
+    "github.com/line/line-bot-sdk-go/v7/linebot/insight"
+    "github.com/line/line-bot-sdk-go/v7/linebot/liff"
+    "github.com/line/line-bot-sdk-go/v7/linebot/manage_audience"
+    "github.com/line/line-bot-sdk-go/v7/linebot/messaging_api"
+    "github.com/line/line-bot-sdk-go/v7/linebot/module"
+    "github.com/line/line-bot-sdk-go/v7/linebot/module_attach"
+    "github.com/line/line-bot-sdk-go/v7/linebot/shop"
+    "github.com/line/line-bot-sdk-go/v7/linebot/webhook"
+)
```
Comment (Yang-33) [inline README.md:43]: for easy copy & paste

File: README.md [hunk 2]
```diff
@@ -65,7 +83,7 @@ import (
 	"github.com/line/line-bot-sdk-go/v7/linebot/webhook"
 )
 
-cb, err := webhook.ParseRequest(req)
+cb, err := webhook.ParseRequest(os.Getenv("LINE_CHANNEL_SECRET"), req)
```
Comment (Yang-33) [inline README.md:86]: actually channel secret is necessary

File: README.md [hunk 3]
```diff
@@ -85,6 +103,16 @@ for _, event := range cb.Events {
...
 
+We provide code [examples](./examples).
+- [EchoBot](./examples/echo_bot/server.go)
+  - a simple echo bot
+- [KitchenSink](./examples/kitchensink/server.go)
+  - a bot that handles many types of events
+- [EchoBotHandler](./examples/echo_bot_handler/server.go)
+- [DeliveryHelper](./examples/delivery_helper/main.go)
+- [InsightHelper](./examples/insight_helper/main.go)
+- [RichmenuHelper](./examples/richmenu_helper/main.go)
```
Comment (Yang-33) [inline README.md:114]: I often hear SDK beginner doesn't know where is sample...

File: README.md (hunk 2 above)
Comment (tokuhirom) [reply README.md:86]: woot

File: linebot/webhook/parse.go [hunk 4]
```diff
@@ -15,7 +15,6 @@ var (
 )
 
 // ParseRequest func
-// Deprecated: Use OpenAPI based classes instead.
```
Comment (tokuhirom) [inline linebot/webhook/parse.go:18]: oops

**Review by tokuhirom — APPROVED** [review]

**Comment by tokuhirom** [comment]
Thanks a lot!!!

**Pull request merged**